backend/shork/src/test/kotlin/compiler/TestParser.kt

```diff
@@ -19,6 +19,28 @@ import software.shonk.interpreter.internal.instruction.Seq
 import software.shonk.interpreter.internal.instruction.Sne
 
 internal class TestParser {
+    @Test
+    fun `parser can parse negative numbers as fields`() {
```

Comment:
A test that checks if parsing of a negative number also works for the b field would be nice, just to make sure it works in both cases